### DISCUSSION TITLE
Adding support for "Privacy Portal" email aliases

### DISCUSSION THREAD

**Discussion by privacyguard** [bitwarden/clients#12338] [Password Manager]
### Code Contribution Proposal

We'd love to add support for [**Privacy Portal**](https://privacyportal.org/#mail-relay) email aliases under "Forwarded Email Aliases" as an additional service focused on privacy and offering features such as:
- Email Encryption (PGP and S/MIME)
- Metadata Leak Protection
- Spam Protection
- In-Memory Processing

![web](https://github.com/user-attachments/assets/98731e2c-cb69-4143-8a2b-c953262b5609)

We are happy to contribute the PRs to web and mobile clients!

**Comment by micahblut** [comment]
Thank you for raising the discussion. Your solution ticks a couple of boxes - being open source and having a free option are big pluses. It's also great to hear you'd be happy to contribute the PRs to our clients, iOS, and Android repositories. I think I'd like to see some organic demand from Bitwarden users before we give a thumbs up to contributions of this sort though. We'll see what others think!
